run_experiment.py

```diff
@@ -8,15 +8,17 @@
 from viz.visualize import Visualizer as Viz    
 
 
-def retrieve_test_data(experiment_name, batch_size=32):
-    """ Get the test set for the relevant experiment..
+def random_samples(experiment_name, num_samples=1):
+    """ generate a number of random samples from the dataset
     """
     with open('trained_models/{}/specs.json'.format(experiment_name)) as spec_file:
         spec_data = json.load(spec_file)
         dataset_name = spec_data['dataset']
 
-    _, test = get_dataloaders(batch_size=batch_size, dataset=dataset_name)
-    return test
+        _, data_loader = get_dataloaders(batch_size=num_samples, dataset=dataset_name, shuffle=True)
```

Comment:
don't you want keep test_loader here to know what data we are talking about ?

Reply:
Sure, I'll make this clear.